README.md

```diff
@@ -148,7 +148,10 @@ The Amazon Chime SDK for JavaScript uses WebRTC, the real-time communication API
 
 ## Installation
 
-Make sure you have Node.js version 12 or higher. Node 14 is recommended and supported.
+Make sure you have the following dependency installed:
+- Node.js version 12 or higher. Node 14 is recommended and supported.
+- Typescript version 4.2.3 (Later version may cause build errors in `lib.d.ts`).
```

Comment:
Suggest changing:
1. Typescript -> TypeScript
2. Later version -> Later versions
3. Would be good to add Ruby version and say "This is needed to run `npm run build` command successfully.

Reply:
I honestly think any Ruby version would work, thus don't put any version there since it is only for shell scripting. Just add v3 for some reference.